Source/NSTextCheckingController.m

```diff
@@ -0,0 +1,137 @@
+/* Implementation of class NSTextCheckingController
+   Copyright (C) 2020 Free Software Foundation, Inc.
+   
+   By: Gregory John Casamento
+   Date: 02-08-2020
+
+   This file is part of the GNUstep Library.
+   
+   This library is free software; you can redistribute it and/or
+   modify it under the terms of the GNU Lesser General Public
+   License as published by the Free Software Foundation; either
+   version 2.1 of the License, or (at your option) any later version.
+   
+   This library is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+   Lesser General Public License for more details.
+   
+   You should have received a copy of the GNU Lesser General Public
+   License along with this library; if not, write to the Free
+   Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+   Boston, MA 02110 USA.
+*/
+
+#import "AppKit/NSTextCheckingController.h"
+#import "AppKit/NSSpellChecker.h"
+
+@implementation NSTextCheckingController
+
+// initializer
+- (instancetype) initWithClient: (id<NSTextCheckingClient>)client
+{
+  self = [super init];
+  if (self != nil)
+    {
+      _client = client;
+      _spellCheckerDocumentTag = 0;
+    }
+  return self;
+}
+
+// properties...
+- (id<NSTextCheckingClient>) client
+{
+  return _client;
+}
+
+- (NSInteger) spellCheckerDocumentTag
+{
+  return _spellCheckerDocumentTag;
+}
+
+- (void) setSpellCheckerDocumentTag: (NSInteger)tag
+{
+  _spellCheckerDocumentTag = tag;
+}
+
+// instance methods...
+- (void) changeSpelling: (id)sender
+{
+}
+
+- (void) checkSpelling: (id)sender
+{
+    int wordCount = 0;
+  [[NSSpellChecker sharedSpellChecker] checkSpellingOfString: [_client string]
```

Comment:
Looks like something with the indentation is wrong here.

Reply:
Corrected, check on your side to make sure.